lyricsgenius/genius.py

```diff
@@ -307,9 +307,10 @@ def search_album(self, name=None, artist="",
                 print('Searching for "{s}"...'.format(s=name))
 
         if album_id:
+            search_term = "id:{id}".format(id=album_id)
```

Comment:
Why set the `search_term` here? It's not going to be used.

Reply:
To avoid a `NameError` in the following print statement in case `self.album(album_id, text_format).get('album')` returns None.
If you're asserting that it will never happen, then the `.get('album')` part should be changed to `['album']`.

Reply:
It'll never happen since it'll result in an HTTP error. Please remove the line and change `.get('album')` to `['album']` as you mentioned.